packages/lib/temporal/client.ts

```diff
@@ -0,0 +1,21 @@
+import { Connection, Client } from "@temporalio/client";
+
+let clientInstance: Client | null = null;
+
+//use a singleton client to avoid creating multiple connections.
```

Comment:
I thought of a better way to inject the Temporal Client connection in the app but I don't understand how trpc works. Perhaps we could use some sort of constructor dependency injection. 

My last resort is to just instantiate a singleton and reuse the object